src/main/scala/tile/LazyRoCC.scala

```diff
@@ -64,7 +62,7 @@ abstract class LazyRoCC(implicit p: Parameters) extends LazyModule {
   val tlNode: TLNode = TLIdentityNode()
 }
 
-class RoCCIO(outer: LazyRoCC)(implicit p: Parameters) extends RoCCCoreIO()(p) {
+class RoCCIO(val outer: LazyRoCC)(implicit p: Parameters) extends RoCCCoreIO()(p) {
```

Comment:
I think this can/should be private. @jackkoenig is that right?

I'm not sure whats even using the member. Perhaps its vestigial?

Reply:
The new `cloneType` doesn't care about public/private, so it's a matter of policy on the part of the device maker. @mwachs5 was advocating for more of them to be `private` for consistency, but there was at least one case in sifive-blocks where it had to be public.

I think you are right this is vestigal but I'm merging it anyway. All of this code should be revisited in light of auto-punching of diplomatic bundles and LazyScopes...